### ISSUE TITLE
IPython adds spurious members to addicts

### ISSUE THREAD

**Issue by ikrase** [mewwts/addict#75]
This isn't really a problem with Addict, but it might deserve a warning or to be a special case. 

IPython touches objects with getattr when working interactively to determine their nature. This means that you get the following: 
```

In [1]: import addict 

In [2]: d = addict.Dict()

In [3]: d
Out[3]: {'_ipython_canary_method_should_not_exist_': {}}
```
due to the way that Addict overloads getattr. 

Is there a chance that we could address this somehow? It's a particular matter because one of my biggest reasons for using Addict is the interactive value.

**Comment by Ohjeah** [comment]
See also #73 and #26

**Comment by Ohjeah** [comment]
You may want to have a look at: https://github.com/hoytak/treedict

**Comment by mewwts** [comment]
Hey guys, 

I think I fixed this.

Check out the #76, and specifically test case on line 93.

**Comment by mewwts** [comment]
#76 is merged and the fix is now live on `pypi` on version 2.0.0. @Ohjeah, could you fix conda? https://github.com/mewwts/addict/releases/tag/v2.0.0

**Comment by Ohjeah** [comment]
Ok sure:
https://github.com/conda-forge/addict-feedstock/pull/4
